geoh5py/data/data_type.py

```diff
@@ -29,13 +29,29 @@
 from .reference_value_map import ReferenceValueMap
 
 if TYPE_CHECKING:
-    from .. import workspace
-    from . import data  # noqa: F401
+    from ..workspace import Workspace
+    from .data import Data  # noqa: F401
+
+COLOR_MAPPING: list[str] = ["linear", "equal_area", "logarithmic", "cdf", "missing"]
```

Comment:
Sounds like the right place for Literal

```suggestion
from typing import get_args, Literal

ColorMapping = Literal["linear", "equal_area", "logarithmic", "cdf", "missing"]
```